fetch.js

```diff
@@ -167,6 +167,11 @@
 
     this._initBody = function(body) {
       this._bodyInit = body
+
+      delete this._bodyText
+      delete this._bodyBlob
+      delete this._bodyFormData
```

Comment:
At second thought, I guess it can't hurt that this step is kept here. And you're right that accessing private properties is better kept close together instead of scattered through multiple function calls.